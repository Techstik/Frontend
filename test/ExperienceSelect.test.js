/* eslint-disable no-undef */
import { mount } from '@vue/test-utils'
import ExperienceSelect from '@/components/experienceselect'

describe('Experience Select', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(ExperienceSelect)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
  test('matches snapshot', () => {
    const wrapper = mount(ExperienceSelect)
    expect(wrapper.element).toMatchSnapshot()
  })
})
