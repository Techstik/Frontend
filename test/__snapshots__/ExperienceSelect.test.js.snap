// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience Select matches snapshot 1`] = `
<div>
  <div
    class="experience-level"
  >
    <a-checkbox
      default-checked="true"
    />
     
    <p>
      Entry-level
    </p>
     
    <a-progress
      percent="33"
      stroke-color="#b4dfe5"
      stroke-width="15"
    />
  </div>
   
  <div
    class="experience-level"
  >
    <a-checkbox />
     
    <p>
      Intermediate
    </p>
     
    <a-progress
      percent="66"
      stroke-color="#f3976b"
      stroke-width="15"
    />
  </div>
   
  <div
    class="experience-level"
  >
    <a-checkbox />
     
    <p>
      Senior
    </p>
     
    <a-progress
      percent="100"
      stroke-color="#161d2d"
      stroke-width="15"
    />
  </div>
</div>
`;
